liveblog.php

```diff
@@ -737,6 +744,115 @@ public static function get_lazyload_entries( $max_timestamp, $min_timestamp ) {
 		return $result;
 	}
 
+	/**
+	 * Get all entries for specific page
+	 *
+	 * @param int $page Requested Page.
+	 * @return array An array of json encoded results
+	 */
+	public static function get_entries_paged( $page, $last_known_entry = false, $id = false ) {
+
+		if ( empty( self::$entry_query ) ) {
+			self::$entry_query = new WPCOM_Liveblog_Entry_Query( self::$post_id, self::key );
+		}
+
+
+		$per_page = WPCOM_Liveblog_Lazyloader::get_number_of_entries();
+
+		$entries  = self::$entry_query->get_all_entries_asc();
```

Comment:
The existing polling api also seems to call the same function.

https://github.com/Automattic/liveblog/blob/master/liveblog.php#L398

In this method, `get_between_timestamps()` calls it. If it were an issue, I would imagine it would be more of an issue on polling?

Do you have any suggestions of how we could handle it differently?

Reply:
Ah okay, for some reason I thought it was new. Obviously not, so this change should be neutral performance-wise. :+1: